src/aics-image-viewer/components/ViewerStateProvider/index.tsx

```diff
@@ -29,11 +30,17 @@ const VIEWER_SETTINGS_CHANGE_HANDLERS: ViewerSettingChangeHandlers = {
     };
   },
   // Render mode: if we're switching to pathtrace, turn off autorotate
-  renderMode: (prevSettings, renderMode) => ({
-    ...prevSettings,
-    renderMode,
-    autorotate: renderMode === RenderMode.pathTrace ? false : prevSettings.autorotate,
-  }),
+  // Also, do not allow pathtrace mode in any mode other than 3D.
+  renderMode: (prevSettings, renderMode) => {
+    if (renderMode === RenderMode.pathTrace && prevSettings.viewMode !== ViewMode.threeD) {
+      return { ...prevSettings };
+    }
```

Comment:
This was previously enforced in the UI by disabling the path trace option, but it's possible to have conflicting URL parameters.